app/helpers/tasklist_helper.rb

```diff
@@ -39,7 +39,9 @@ def subsection_link(subsection, log, current_user)
 
   def review_log_text(log)
     if log.collection_period_open?
-      "You can #{govuk_link_to 'review and make changes to this log', review_lettings_log_path(log)} until #{log.form.end_date.to_formatted_s(:govuk_date)}.".html_safe
+      link = log.is_a?(SalesLog) ? review_sales_log_path(id: log, sales_log: true) : review_lettings_log_path(log)
```

Comment:
```suggestion
      link = log.sales ? review_sales_log_path(id: log) : review_lettings_log_path(log)
```

Why do you need the `sales_log: true` here?

Reply:
sales?* presumably in the suggested improvement

Reply:
@natdeanlewissoftwire Good spot! Thanks - updated my suggestion.